### PR TITLE
cifs-utils: Update to 6.7

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @ffainelli 
Compile tested: mt7621, LEDE trunk
Run tested: mt7621, LEDE trunk

### CODE REVIEW

**Comment by neheb** [comment]
@ffainelli ping

File: net/cifs-utils/Makefile [hunk 1]
```diff
@@ -8,23 +8,25 @@
...
 
+PKG_FIXUP:=libtool
+
 include $(INCLUDE_DIR)/package.mk
 
 define Package/cifsmount
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-fs-cifs
+  DEPENDS:=+kmod-fs-cifs +libtalloc
```
Comment (pprindeville) [inline net/cifs-utils/Makefile]: Was this always a dependency, but it's only being called out now?  Or do newer versions introduce this dependency?

**Comment by neheb** [comment]
I'm guessing the latter.

File: net/cifs-utils/Makefile (hunk 1 above)
Comment (pprindeville) [reply net/cifs-utils/Makefile]: Can you try building it without this?  What happens if you do?

**Comment by neheb** [comment]
Fails.  Configure scripts looks for it.

**Comment by ffainelli** [comment]
How much bigger is this new cifs-utils + dependencies?

File: net/cifs-utils/Makefile (hunk 1 above)
Comment (pprindeville) [reply net/cifs-utils/Makefile]: Okay, asked and answered... it fails.

**Comment by neheb** [comment]
Old - -rw-r--r-- 1 mangix mangix 12235 Dec  2 14:53 packages/cifsmount_6.4-2_mipsel_24kc.ipk
New - -rw-r--r-- 1 mangix mangix 12185 Dec 13 19:43 packages/cifsmount_6.7-1_mipsel_24kc.ipk
-rw-r--r-- 1 mangix mangix 10277 Dec 13 19:37 packages/libtalloc_2.1.7-2_mipsel_24kc.ipk

compiled on ramips

**Pull request merged**